config/constants.go

```diff
@@ -70,6 +72,8 @@ func setupDevelopmentConstants() {
 	Constants.DbDatabase = "tf2stadium"
 	Constants.DbUsername = "tf2stadium"
 	Constants.DbPassword = "dickbutt" // change this
+
+	Constants.SteamDevApiKey = "your steam dev api key"
```

Comment:
Can you add `overrideFromEnv(&Constants.SteamDevApiKey, "STEAM_API_KEY")` next to other other overrides?